src/components/MediaSettings/MediaSettings.vue

```diff
@@ -431,16 +436,20 @@ export default {
 				this.audioOn = !BrowserStorage.getItem('audioDisabled_' + this.token)
 				this.videoOn = !BrowserStorage.getItem('videoDisabled_' + this.token)
 				this.silentCall = !!BrowserStorage.getItem('silentCall_' + this.token)
-
-				// Set virtual background depending on BrowserStorage's settings
-				if (BrowserStorage.getItem('virtualBackgroundEnabled_' + this.token) === 'true') {
-					if (BrowserStorage.getItem('virtualBackgroundType_' + this.token) === VIRTUAL_BACKGROUND.BACKGROUND_TYPE.BLUR) {
-						this.blurVirtualBackground()
-					} else if (BrowserStorage.getItem('virtualBackgroundType_' + this.token) === VIRTUAL_BACKGROUND.BACKGROUND_TYPE.IMAGE) {
-						this.setVirtualBackgroundImage(BrowserStorage.getItem('virtualBackgroundUrl_' + this.token))
-					}
+				// Check main blur background setting
+				if (this.blurBackgroundEnabled) {
```

Comment:
That should not override value from BrowserStorage (e.g. if I set an Image for this conversation)

Reply:
When the setting is "Off", "None" should be the default.
When the setting is "On", "Blur" should be the default.
If the user changed the selection in an individual room, the Blur, Uploaded image or Select default background should be remembered for the future.
If the user has the setting "On" and manually selects "None", it is okay to ignore this in the next call and have it "Blur" again (due to deleting the entry from the browser storage)

Reply:
> When the setting is "Off", "None" should be the default.

This will affect people who never used the global feature, or should I limit it to those who turned it on and off?

> If the user changed the selection in an individual room, the Blur, Uploaded image or Select default background should be remembered for the future.

By future, do you mean only next call or for all future call?

Reply:
1. When the setting is 'Off', it should behave as before PR (none is default, blur/image can be remembered per conv);
2. for all future calls in this conversation